### PR TITLE
GH-683: Support multiple if conditions in list comprehensions

### LINKED ISSUE #683: Support for if conditions in list comprehensions
Currently VOC supports `list` (and also `set`, `dict`, `generator`) comprehensions, but ignores any `if` conditions present.

**Excepted result:**
```py
>>> [i for i in range(10) if i % 2 == 0]
[0, 2, 4, 6, 8]
```

**Current result in VOC:**
```py
>>> [i for i in range(10) if i % 2 == 0]
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
```

I am interested in working on a fix, but would need some pointers.

### PR DESCRIPTION
**Pull request by onyb**
Refer to PR #686 which added support for a single if-condition. This PR is an extension of that, bringing support for multiple if-conditions in a list comprehension.

Resolves #683.

### CODE REVIEW

File: voc/python/ast.py [hunk 1]
```diff
@@ -1244,7 +1244,9 @@ def visit_ListComp(self, node):
                 self.visit(generator.target)
 
         if node.generators[0].ifs:
-            self.visit(node.generators[0].ifs[0])
+            self.visit(
+                ast.BoolOp(ast.And(), node.generators[0].ifs)
```
Comment (eliasdorneles) [inline voc/python/ast.py]: beautiful! :heart_eyes:

File: tests/structures/test_list_comprehension.py [hunk 2]
```diff
@@ -12,8 +12,8 @@ def test_syntax(self):
 
     def test_list_comprehension_with_if_condition(self):
         self.assertCodeExecution("""
-            x = [1, 2, 3, 4, 5]
-            print([v**2 for v in x if v % 2 == 0])
+            print([v for v in range(100) if v % 2 == 0])
+            print([v for v in range(100) if v % 2 == 0 if v % 3 == 0])
```
Comment (eliasdorneles) [inline tests/structures/test_list_comprehension.py]: Could you add some tests here for more than 2 conditions, just to increase confidence in the test suite? (e.g. adding `if v > 10`, `if v < 80`)

Comment (onyb) [reply tests/structures/test_list_comprehension.py]: Done!

**Comment by eliasdorneles** [comment]
Thanks @onyb ! :100:

**Pull request merged**